src/transformers/models/vit/feature_extraction_vit.py

```diff
@@ -0,0 +1,140 @@
+# coding=utf-8
+# Copyright Google AI and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Feature extractor class for ViT."""
+
+from typing import List, Union
+
+import numpy as np
+import torch
+from PIL import Image
+from torchvision.transforms import Compose, Normalize, Resize, ToPILImage, ToTensor
+
+from ...feature_extraction_utils import BatchFeature, FeatureExtractionMixin
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+
+class ViTFeatureExtractor(FeatureExtractionMixin):
+    r"""
+    Constructs a ViT feature extractor.
+
+    This feature extractor inherits from :class:`~transformers.FeatureExtractionMixin` which contains most of the main
+    methods. Users should refer to this superclass for more information regarding those methods.
+
+    Args:
+        image_mean (:obj:`int`, defaults to [0.5, 0.5, 0.5]):
+            The sequence of means for each channel, to be used when normalizing images.
+        image_std (:obj:`int`, defaults to [0.5, 0.5, 0.5]):
+            The sequence of standard deviations for each channel, to be used when normalizing images.
+        do_normalize (:obj:`bool`, `optional`, defaults to :obj:`True`):
+            Whether or not to normalize the input with mean and standard deviation.
+        do_resize (:obj:`bool`, `optional`, defaults to :obj:`True`):
+            Whether to resize the input to a certain :obj:`size`.
+        size (:obj:`int`, `optional`, defaults to :obj:`224`):
```

Comment:
```suggestion
        size (:obj:`int`, `optional`, defaults to 224):
```
No obj for ints or floats.